integrated_channels/canvas/transmitters/content_metadata.py

```diff
@@ -21,6 +21,9 @@ def __init__(self, enterprise_configuration, client=CanvasAPIClient):
         )
 
     def _prepare_items_for_transmission(self, channel_metadata_items):
+        # here is a hack right now to send only one item
+        # we have to investigate how to handle multiple
+        # metadata items
         return {
-            'course': channel_metadata_items,
+            'course': channel_metadata_items[0],
```

Comment:
ah this is the work around. Makes sense.

Reply:
yup indeed, now trying to see what happens with multiple metadata pieces, if it just works, then we can leave it be

Reply:
I think I'd be more concerned if this did work on multiple metadata dicts lol

Reply:
yeah so I just tested with two metadata content pieces and it picked up both and sent one at a time and I had two courses ... 

```
2020-08-18 17:11:21,126 INFO 1108 [integrated_channels.integrated_channel.transmitters.content_metadata] [user None] content_metadata.py:79 - Preparing to transmit creation of [2] content metadata items with plugin configuration [<CanvasEnterpriseCustomerConfiguration for Enterprise Test Enterprise>]: [['course-v1:canvas-test-edx+ct102+ct102_T1', 'c900d2b6-d59d-435d-8bff-c8c4a0ee3781']]
```

Reply:
oh right I forgot it sends them one at a time. That's fine then. I was a bit concerned there.